libs/ui-lib/lib/common/components/ui/OpenShiftVersionDropdown.tsx

```diff
@@ -90,7 +90,7 @@ export const OpenShiftVersionDropdown = ({
 
   const parsedVersionsForCustomItems = getParsedVersions(customItems);
   const customDropdownItems = parsedVersionsForCustomItems.parsedVersions.map(({ y, versions }) => {
-    const customItems = versions.map(({ value, label }) => (
+    const customItems = versions.reverse().map(({ value, label }) => (
```

Comment:
It's not efficient to run the sorting on each render, in every place where the versions are used. Plus by doing a reverse, a future change on the API or somewhere else in the UI could easily break it.

I think either:

    Apply the desired sorting algorithm in useOpenshiftVersions.sortVersions. Then IIUC, when you rebuild the option groups you'd keep this order so there wouldn't be a need to do a new sorting.

    If the above is not possible, then it's preferable to sort once in getParsedVersions. I would still avoid sorting "blindly" using reverse, instead by using a custom sorting function that guarantees that you get the desired order.

Hopefully by doing the change in 1, this would correctly apply to all the places that show the versions.

Reply:
I've changed to do it in parsedVersions function